skimage/restoration/_denoise.py

```diff
@@ -258,9 +258,9 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
 
 
 @utils.channel_as_last_axis()
-@utils.deprecate_multichannel_kwarg()
+@utils.deprecate_multichannel_kwarg(extra_stacklevel=1)
 @utils.deprecate_kwarg({'max_iter': 'max_num_iter'}, removed_version="1.0",
-                       deprecated_version="0.19")
+                       deprecated_version="0.19", extra_stacklevel=1)
```

Comment:
```suggestion
                       deprecated_version="0.19")
```
This is causing the CI failures: `extra_stacklevel` is defined for `deprecate_kawarg`